cmd/nerdctl/inspect.go

```diff
@@ -42,18 +42,35 @@ func newInspectCommand() *cobra.Command {
 	return inspectCommand
 }
 
+var validInspectType = map[string]bool{
+	"container": true,
+	"image":     true,
+	"all":       true,
```

Comment:
Not sure we want `all`. Not present in Docker.

Reply:
I thought the same, but this way we also preserve the way nerdctl works currently, what do you think?
Will obviously fix the previous comment, defaulting to `container`

Reply:
I think the current way was wrong

Reply:
No problem then, I'll remove the "all" option :+1: